src/protocol/mod.rs

```diff
@@ -249,6 +249,7 @@ impl Protocol {
 pub struct Protocols {
     pub protocols: Vec<Protocol>,
     pub unmocked_commands: Vec<Vec<u8>>,
+    pub interpreter: Option<Vec<u8>>,
```

Comment:
This is the new field added to `Protocols` used to represent the specified interpreter, if given